src/stories/containers/Finances/components/ConditionalWrapper/ConditionalWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from '@emotion/styled';
+import Container from '@ses/components/Container/Container';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+
+interface Props extends React.PropsWithChildren {
+  period: string;
+}
+
+const ConditionalWrapper: React.FC<Props> = ({ children, period }) => {
+  const Wrapper =
+    period === 'Quarterly' ? <ContainerTable>{children}</ContainerTable> : <Container>{children}</Container>;
+  return Wrapper;
```

Comment:
you could return it directly to avoid unnecessary variables (which should be in lower case btw due name conventions)

Reply:
you are right, this is something that this functionality is show betters when add other header, than will made sense the condition
Done